app/src/main/java/net/skyscanner/backpack/demo/ComponentDetailActivity.kt

```diff
@@ -52,6 +52,7 @@ class ComponentDetailActivity : BpkBaseActivity() {
       // Create the detail fragment and add it to the activity
       // using a fragment transaction.
       val itemId = intent.getStringExtra(ComponentDetailFragment.ARG_ITEM_ID)
+        ?: error("ComponentDetailActivity intent must have ${ComponentDetailFragment.ARG_ITEM_ID} set")
```

Comment:
Nullability information was added to Intent.getStringExtra() in combileSdk=30. Therefore we need to do a nullability check here.